x/evm/keeper/grpc_query.go

```diff
@@ -533,13 +541,85 @@ func (k Keeper) TraceBlock(c context.Context, req *types.QueryTraceBlockRequest)
 	}, nil
 }
 
-// traceTx do trace on one transaction, it returns a tuple: (traceResult, nextLogIndex, error).
-func (k *Keeper) traceTx(
+// TraceCall configures a new tracer according to the provided configuration, and
+// executes the given call in the provided environment. The return value will
+// be tracer dependent.
+func (k Keeper) TraceCall(c context.Context, req *types.QueryTraceCallRequest) (*types.QueryTraceCallResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	if req.TraceConfig != nil && req.TraceConfig.Limit < 0 {
```

Comment:
not sure if we need handle [stateOverrides](https://github.com/ethereum/go-ethereum/blob/v1.10.26/eth/tracers/api.go#L847)

Reply:
I can see in `eth_call` the state overrides param are ignored too (https://github.com/crypto-org-chain/ethermint/blob/199ccc232fd4cc5b0468b0cdec5e1ddaa340e116/rpc/namespaces/ethereum/eth/api.go#L283). But not sure about the context @yihuang do you have any insight on this?

In addition, from the `debug_traceCall` API documentation, I could not see if can specify state override though?

Reply:
seems like some new feature introduced in go-ethereum, we don't support yet.